### PR TITLE
APEXMALHAR-2547 nyc taxi example checkin

### PR DESCRIPTION
**Pull request by davidyan74**
@tweise please take a look. thanks.

### CODE REVIEW

**Comment by tweise** [comment]
JIRA please.

**Comment by tweise** [comment]
rat:check isn't happy about missing license headers.

File: library/src/main/java/com/datatorrent/lib/io/WebSocketInputOperator.java [hunk 1]
```diff
@@ -207,7 +207,6 @@ public Thread newThread(Runnable r)
         @Override
         public void onMessage(String string)
         {
-          LOG.debug("Got: " + string);
```
Comment (pramodin) [inline library/src/main/java/com/datatorrent/lib/io/WebSocketInputOperator.java:210]: An accidental removal of the debug message or was it intentional?

Comment (davidyan74) [reply library/src/main/java/com/datatorrent/lib/io/WebSocketInputOperator.java:210]: I thought we should remove it but I can put it back if you insist.

**Comment by davidyan74** [comment]
@tweise what should I do re: the license headers in csv? csv does not support comment.

File: library/src/main/java/com/datatorrent/lib/io/WebSocketInputOperator.java (hunk 1 above)
Comment (pramodin) [reply library/src/main/java/com/datatorrent/lib/io/WebSocketInputOperator.java:210]: No strong opinion either way just wanted to check it was not accidental removal. Wouldn't having the ability to see the received message be useful for debugging or are there other ways to get to it.

File: examples/nyctaxi/src/main/java/org/apache/apex/examples/nyctaxi/Application.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import com.google.common.base.Throwables;
+
+import com.datatorrent.api.DAG;
+import com.datatorrent.api.StreamingApplication;
+import com.datatorrent.api.annotation.ApplicationAnnotation;
+import com.datatorrent.lib.io.ConsoleOutputOperator;
+import com.datatorrent.lib.io.PubSubWebSocketAppDataQuery;
+import com.datatorrent.lib.io.PubSubWebSocketAppDataResult;
+
+/**
```
Comment (tweise) [inline examples/nyctaxi/src/main/java/org/apache/apex/examples/nyctaxi/Application.java:45]: please remove

**Comment by tweise** [comment]
@davidyan74 you may have to add the *.csv exclusion to the top level pom.xml for both, the license and rat plugins. So far it only occurred in test resources.

File: examples/nyctaxi/src/main/java/org/apache/apex/examples/nyctaxi/Application.java (hunk 2 above)
Comment (davidyan74) [reply examples/nyctaxi/src/main/java/org/apache/apex/examples/nyctaxi/Application.java:45]: Done.

**Comment by davidyan74** [comment]
Added csv exclusion. Thanks.

**Comment by tweise** [comment]
see #677 for test failure

**Comment by davidyan74** [comment]
@tweise Thanks. Will pull the latest when #677 is merged.

**Comment by tweise** [comment]
@davidyan74 rebase

**Comment by davidyan74** [comment]
@tweise Looks like the flume sink test is still stalling.

**Comment by davidyan74** [comment]
@tweise Hmmm. Looks like the test is flaky.

**Pull request merged**